### ISSUE TITLE
the getter _key was called on null

### ISSUE THREAD

**Issue by FilledStacks** [jonataslaw/get#105]
**Describe the bug**
When setting the navigation for the router to Get.key it throws a null exception

**To Reproduce**
Clone and run the [example here](https://github.com/FilledStacks/stacked/tree/master/packages/stacked_services) on the latest Flutter 1.17.1 

**Expected behavior**
When setting the key it should not be throwing a null error

**Flutter Version:**
1.17.1

**Get Version:**
2.5.0

**Describe on which device you found the bug:**
Emulator, all

**Minimal reproduce code**
https://github.com/FilledStacks/stacked/tree/master/packages/stacked_services

**Comment by FilledStacks** [comment]
Fixed in my package by adding.

```dart
  get navigatorKey {
    // We construct this instance to make sure the _get value inside the Get package is not null
    var tempConstruction = Get();
    return Get.key;
  }
```

**Comment by jonataslaw** [comment]
> Fixed in my package by adding.
> 
> ```dart
>   get navigatorKey {
>     // We construct this instance to make sure the _get value inside the Get package is not null
>     var tempConstruction = Get();
>     return Get.key;
>   }
> ```

Hi, thanks for reporting this, first of all, congratulations on your channel, it is undoubtedly one of the best channels on Flutter in the world.
As for the issue, in the penultimate update I removed all static variables from the code, leaving only get static methods for some essential services, and perhaps, making the navigatorKey private and internal has required that an instance be created to start it.
NavigatorKey will be present from the start up, until the end of the app, so you may have no problem using it statically. I did not foresee this situation. Thanks for reporting I will fix this now.

**Comment by FilledStacks** [comment]
@jonataslaw thank you very much for those kind words, and thank you very much for building this package as well. I was very close to wrapping my "contextless" navigation pattern into a package as well that would have ultimately been my NavigationService, now I just use your package for everything context related :) 

It has a lot of other very nice features as well. I'm going to be exploring it more after I build the 3 apps currently in development. 

Thanks for the quick response.

**Comment by jonataslaw** [comment]
Fix on 2.5.2

I tried to start the example of your package on Flutter_web to check if there were errors in the new version, and I saw that it didn't have web / desktop support.
Fix a PR there to support all platforms, nothing too critical, just removed the references to dart: io.

Closing this issue, if you face any other problem, do not hesitate to open another issue.
